src/aho-corasick.cpp

```diff
@@ -426,6 +427,22 @@ This function performs the same as ``traverse_word(ac, AhoCorasick.root, w)``
 Traverse the trie of *ac* from the root using suffix links where necessary.
 
 This function performs the same as ``traverse_word(ac, AhoCorasick.root, w)``
+)pbdoc");
+
+    m.def(
+        "aho_corasick_dot",
+        [](AhoCorasick& ac) { return aho_corasick::dot(ac); },
+        py::arg("ac"),
+        R"pbdoc(
+:sig=(ac: AhoCorasick) -> Dot:
+::only-document-once:
```

Comment:
```suggestion
```
We don't need this here, since nothing is being overloaded

Reply:
Tah, forgot to push :)